scripts/ci/run_perftest.sh

```diff
@@ -119,125 +139,153 @@ force_compaction() {
 
   set -e
   for shard in $shards; do
-    /home/ubuntu/influx_tools compact-shard -force -verbose -path $shard
+    if [ -n "$(ls -A $shard)" ]; then
```

Comment:
Is this making it so it won't try to compact folders without any TSM files?

Reply:
Yes

Reply:
Though I changed the way I'm checking... see latest code